portal/src/main/webapp/WEB-INF/jsp/visualize.jsp

```diff
@@ -38,7 +38,14 @@
 <jsp:include page="global/header.jsp" flush="true" />
 
 <script>
+
+if (/isolateNetwork=1/.test(window.location.search)) {
```

Comment:
i guess this is okay although it does feel hacky..

Reply:
oh this is a whole thing is a glorious hack.  i think the above is ok.  it's in a throwaway codebase.